resources/user-agents/apps/itunes/itunes-1-2-to-4-7.json

```diff
@@ -31,7 +31,7 @@
           "match": "iTunes/#MAJORVER#.#MINORVER#* (#PLATFORM#)*",
           "device": "Windows Desktop",
           "platforms": [
-            "Windows_64",
+            "Windows_x64",
```

Comment:
These were the only UAs that used the "Windows_64" platform, I just changed its name to better match the `WOW64` vs `x64` platforms.